### PR TITLE
[ENH] Add sys info for mne-icalabel

### PR DESCRIPTION
**Pull request by adam2392**
#### Reference issue
Closes: #10609 

#### What does this implement/fix?
Explain your changes.

#### Additional information
Any additional information you think is important.

### CODE REVIEW

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -23,6 +23,9 @@ Current (1.1.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
+
+- Add ``mne-icalabel`` to :func:`mne.sys_info` (:gh:`10615` by `Adam Li`_)
```
Comment (larsoner) [inline doc/changes/latest.inc:27]: the `:newcontrib:` should go at the top, can you move this down?

Comment (larsoner) [reply doc/changes/latest.inc:27]: I'll do this in #10592 to save one of us a rebase

**Comment by larsoner** [comment]
Thanks @adam2392 !

File: doc/changes/latest.inc (hunk 1 above)
Comment (adam2392) [reply doc/changes/latest.inc:27]: Ah thanks!

**Pull request merged**